spec/vast_trackers.spec.js

```diff
@@ -282,5 +285,34 @@ describe('VASTTracker', function() {
         expect(spyTrack).not.toHaveBeenCalledTimes(2);
       });
     });
+    describe('#error', () => {
+      isCustomCode = false;
+      beforeEach(() => {
+        vastTracker.error();
```

Comment:
As there is only one `it` inside this `describe` can we move the error method call inside the `it` ? Also as we use the variable isCustomCode only once IMO we can remove it and do `{isCustomCode: false}` l.297